src/Datadog.Trace.ClrProfiler.Managed/Helpers/WebHelpers.cs

```diff
@@ -0,0 +1,19 @@
+namespace Datadog.Trace.ClrProfiler
+{
+    internal static class WebHelpers
```

Comment:
Consider naming this class `SpanExtensions` to follow C# conventions.

Reply:
Moved to SpanExtensions file